src/framework/ui/modal/modal.component.tsx

```diff
@@ -27,7 +24,7 @@ interface ModalProps {
 
 const { width, height } = Dimensions.get('window');
 
-export type Props = StyledComponentProps & ViewProps & ModalProps;
+export type Props = ViewProps & ModalProps;
```

Comment:
Is it not styled anymore?

Reply:
You're right, modal is not a part of our system. It's just a component for using custom content.